azure-page/src/App.js

```diff
@@ -0,0 +1,48 @@
+import logo from "./logo.svg";
+import img from "./manage.png";
+import "./index.css";
+// import Dropdown from "./components/dropdown/explore/explore";
+import Header from "./components/header";
+import Banner from "./components/banner";
+import Card from "./components/Card";
+// import Explore from "./components/dropdown/explore/explore";
+// import DropDown from "./components/dropdown/explore/DropdownMain";
+
+const title = "Azure";
+const link1 = "https://microsoft.com/";
+const link2 = "https://azure.microsoft.com/en-gb/";
+
+
+function App() {
+  return (
+    <div className="App container">
```

Comment:
Follow CSS convention for classname naming.
Should be "app"

all lower case separated by dashes. 
 Follow
kebab-case. Also called caterpillar-case, dash-case, hyphen-case, lisp-case, spinal-case and css-case

Reply:
Resolved